### PR TITLE
Generate naming for OWNERS review

### PR DESCRIPTION
**Pull request by smarterclayton**
This is the mapping of namespace/name -> tag -> source config for CI

OWNERS files are seeded at the appropriate levels. The top level
naming would be centrally controlled with teams delegated.

Example README included.

Mirroring generation included but not wired up.

This should be merged immediately after changes to prowgen.

### CODE REVIEW

File: ci-operator/promotions/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+The directory describes the contents of all published images generated by ci-operator jobs after code is merged to a repository branch under CI control.
```
Comment (stevekuznetsov) [inline ci-operator/promotions/README.md]: If this is going to mostly result in people messing with `ci-operator` configuration, I'd prefer this to be a help page or (sub-page of the ci-operator overview) on the step registry website. Ideally we're getting to a place where someone has a one-stop-shop for everything from "I'm onboarding a repo, what does that mean" to a full integration with CI

Comment (smarterclayton) [reply ci-operator/promotions/README.md]: I.e. including our processes?  That's fine, I just wanted to have it written out to link to the example.

Comment (stevekuznetsov) [reply ci-operator/promotions/README.md]: Yes.

**Comment by smarterclayton** [comment]
/hold

has to be merged with the other PR

**Pull request closed without merging**